bot/resources/valentines/zodiac_explanation.json

```diff
@@ -0,0 +1,122 @@
+    "full_form": "__**V**__irtuous\n__**I**__ntelligent\n__**R**__esponsible\n__**G**__enerous\n__**O**__ptimistic",
+    "url": "https://www.horoscope.com/images-US/signs/profile-virgo.png"
+  },
+  "Libra": {
+    "start_at": "2020-09-23",
+    "end_at": "2020-10-22",
+    "About": "Amazing people born between **September 23** to **October 22**. Libra is an air sign represented by the scales (interestingly, the only inanimate object of the zodiac), an association that reflects Libra's fixation on balance and harmony. Libra is obsessed with symmetry and strives to create equilibrium in all areas of life.",
+    "Motto": "***\u201cNo person is an island.\u201d***",
+    "Strengths": "cooperative, diplomatic, gracious, fair-minded, social.",
+    "Weaknesses": "indecisive, avoids confrontations, will carry a grudge, self-pity.",
+    "full_form": "__**L**__oyal\n__**I**__nquisitive\n__**B**__alanced\n__**R**__esponsible\n__**A**__ltruistic",
+    "url": "https://www.horoscope.com/images-US/signs/profile-libra.png"
+  },
+  "Scorpio": {
+    "start_at": "2020-10-23",
+    "end_at": "2020-11-21",
+    "About": "Amazing people born between **October 23** to **November 21**. Scorpio is one of the most misunderstood signs of the zodiac. Because of its incredible passion and power, Scorpio is often mistaken for a fire sign. In fact, Scorpio is a water sign that derives its strength from the psychic, emotional realm.",
+    "Motto": "***\u201cYou never know what you are capable of until you try.\u201d***",
+    "Strengths": "resourceful, brave, passionate, stubborn, a true friend.",
+    "Weaknesses": "distrusting, jealous, secretive, violent.",
+    "full_form": "__**S**__eductive\n__**C**__erebral\n__**O**__riginal\n__**R**__eactive\n__**P**__assionate\n__**I**__ntuitive\n__**O**__utstanding",
+    "url": "https://www.horoscope.com/images-US/signs/profile-scorpio.png"
+  },
+  "Sagittarius": {
+    "start_at": "2020-11-22",
+    "end_at": "2020-12-21",
+    "About": "Amazing people born between **November 22** to **December 21**. Represented by the archer, Sagittarians are always on a quest for knowledge. The last fire sign of the zodiac, Sagittarius launches its many pursuits like blazing arrows, chasing after geographical, intellectual, and spiritual adventures.",
+    "Motto": "***\u201cTowering genius disdains a beaten path.\u201d***",
+    "Strengths": "generous, idealistic, great sense of humor.",
+    "Weaknesses": "promises more than can deliver, very impatient, will say anything no matter how undiplomatic.",
+    "full_form": "__**S**__eductive\n__**A**__dventurous\n__**G**__rateful\n__**I**__ntelligent\n__**T**__railblazing\n__**T**__enacious adept\n__**A**__dept\n__**R**__esponsible\n__**I**__dealistic\n__**U**__nparalled\n__**S**__ophisticated",
+    "url": "https://www.horoscope.com/images-US/signs/profile-sagittarius.png"
+  },
+  "Capricorn": {
+    "start_at": "2020-12-22",
+    "end_at": "2020-01-19",
```

Comment:
the year should be 2021 here, because it doesn't make sense to go from 2020 December to 2020 Jan.

Reply:
Ok